data/json/items/tool_armor.json

```diff
@@ -60,6 +60,44 @@
     "techniques": [ "WBLOCK_1" ],
     "magazine_well": 1
   },
+  {
+    "id": "welding_mask",
+    "type": "ARMOR",
+    "name": "welding mask",
+    "description": "A steel mask with a very dark visor, meant to protect the entire face while welding.  It also serves as good armor, but makes it hard to see.  Activate it to push it up your head.",
+    "weight": 4000,
+    "volume": 4,
```

Comment:
```suggestion
    "volume": "1 L",
```

Reply:
This should probably be more like 4 L too.

Reply:
```suggestion
    "volume": "4 L",
```